documentation/src/docs/deployment/kubernetes-install.md

```diff
@@ -0,0 +1,378 @@
+# Deploying in Kubernetes
+
+## Table of Contents
+
+ * [Requirements](#requirements)
+    * [Pravega Operator](#pravega-operator)
+ * [Usage](#usage)    
+    * [Installation of the Pravega Operator](#install-the-pravega-operator)
+    * [Deploy a sample Pravega Cluster](#deploy-a-sample-pravega-cluster)
+    * [Uninstall the Pravega Cluster](#uninstall-the-pravega-cluster)
+    * [Uninstall the Pravega Operator](#uninstall-the-pravega-operator)
+ * [Configuration](#configuration)
+    * [Use non-default service accounts](#use-non-default-service-accounts)
+    * [Installing on a Custom Namespace with RBAC enabled](#installing-on-a-custom-namespace-with-rbac-enabled)
+    * [Tier 2: Google Filestore Storage](#use-google-filestore-storage-as-tier-2)
+    * [Tune Pravega Configurations](#tune-pravega-configuration)
+* [Releases](#releases)
+
+## Requirements
+
+- Kubernetes 1.8+
+- An existing Apache Zookeeper 3.5 cluster. This can be easily deployed using our [Zookeeper Operator](https://github.com/pravega/zookeeper-operator).
+- [Pravega Operator](https://github.com/pravega/pravega-operator/edit/master/README.md) manages Pravega clusters deployed to Kubernetes and automates tasks related to operating a Pravega cluster.
+
+## Usage
+
+### Install the Pravega Operator
+
+> Note: If you are running on Google Kubernetes Engine (GKE), please [check this first](#installation-on-google-kubernetes-engine).
```

Comment:
Point that link to the GitHub repository.

Reply:
Edited